sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -478,11 +479,12 @@ Table {counter: tableNumber}. Constructor of the `command_graph` class.
 |
 [source,c++]
 ----
-command_graph(const device& syclDevice, const property_list& propList = {});
+command_graph(const context& syclContext, const device& syclDevice, const
+property_list& propList = {});
```

Comment:
Slight preference for indenting the parameter
```suggestion
command_graph(const context& syclContext, const device& syclDevice,
              const property_list& propList = {});
```

Reply:
Fixed!